nearlib/test/integration.test.js

```diff
@@ -231,7 +231,7 @@ describe('with deployed contract', () => {
         await expect(contract.triggerAssert()).rejects.toThrow(/Transaction .+ failed.+expected to fail/);
         expect(logs.length).toBe(3);
         expect(logs[0]).toEqual(`[${contractName}]: LOG: log before assert`);
-        expect(logs[1]).toMatch(new RegExp(`^\\[${contractName}\\]: ABORT: "expected to fail" filename: "main.ts" line: \\d+ col: \\d+$`));
+        expect(logs[1]).toMatch(new RegExp(`^\\[${contractName}\\]: ABORT: "expected to fail" filename: "(../out/)?main.ts" line: \\d+ col: \\d+$`));
```

Comment:
Shouldn't it always be `../out/main.ts` after this change?

Reply:
I suppose it would